scripts/package/tools/build

```diff
@@ -0,0 +1,100 @@
+#! /usr/bin/env bash
+
+# Builds a specific package in the context of FPM's Docker container.
+# https://fpm.readthedocs.io/
+
+# Enforce preconditions
+set -ex
+version="$1"
+install_dir=/usr/lib/pash
+set +x -u; : "$version"
+shift # for remaining $@
+
+
+print_dotfpm() {
+    output_format="$1"
+    output_file="$2"
+    
+    # I/O rules, with Docker container paths
+    flag --chdir /src
+    flag --input-type dir
+    flag --output-type "$output_format"
+    flag --package "$output_file"
+    flag --log info
+    flag --verbose
+    flag --force
+
+    # Discovery information
+    flag --name pash
+    flag --description "Parallelize POSIX shell scripts"
+    flag --license mit
+    flag --architecture all
+    flag --url 'https://github.com/binpash/pash'
+    flag --version "$version"
+
+
+    # System-level dependencies
+    flag --depends automake
+    flag --depends bash
+    flag --depends bc
+    flag --depends bsdmainutils
+    flag --depends curl
+    flag --depends git
+    flag --depends g++-10
+    flag --depends libffi-dev
+    flag --depends libtool
+    flag --depends locales
+    flag --depends locales-all
+    flag --depends m4
+    flag --depends netcat-openbsd
+    flag --depends pkg-config
+    flag --depends python # Redundant?
+    flag --depends python3
+    flag --depends python3-pip
+    flag --depends python3-setuptools
+    flag --depends python3-testresources
+    flag --depends wamerican-insane
+    flag --depends wget
+
+    # Handle requirements.txt with `python3 -m pip`
+    flag --python-bin /bin/python3
+    flag --python-obey-requirements-txt
+    flag --python-internal-pip
```

Comment:
These flags were not initially useful. FPM seemed somehow "blind" to `requirements.txt`, so I needed to repeat the same Python package ritual in post-install. Not sure yet if that's a bug in FPM, or me using it incorrectly.

Reply:
I misunderstood: These flags are active when you use FPM to package PaSh _as_ a Python package.